api-reference/50 Common/utils/ui/dialog/custom(options).md

```diff
@@ -36,8 +36,6 @@ Specifies whether to show the title. Defaults to **true**.
 The dialog's title.
 
 ---
-The dialog's message can contain HTML markup. Make sure this markup is secure. 
-
 The **custom(options)** method only creates a dialog. To display it, call the dialog instance's **show()** method. This method returns a Promise. To identify the clicked button, implement a Promise fulfillment handler. The dialog result is passed to the handler and contains anything you return from the clicked button's [onClick](/api-reference/10%20UI%20Components/dxButton/1%20Configuration/onClick.md '/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick') function. 
```

Comment:
```suggestion
The **custom(options)** method only creates a dialog. To display it, call the dialog instance's **show()** method. This method returns a Promise. To identify the clicked button, implement a Promise fulfillment handler. The resulting dialog is passed to the handler and contains anything you return from the clicked button's [onClick](/api-reference/10%20UI%20Components/dxButton/1%20Configuration/onClick.md '/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick') function. 
```

Reply:
By "dialog result" I meant "Yes" or "No", "OK" or "Cancel", etc.

Reply:
If that's a technical term, then ok.